app/src/main/java/com/learnig/android/cleanarchitecture/core/di/AppModule.kt

```diff
@@ -4,11 +4,12 @@ import androidx.recyclerview.widget.DividerItemDecoration
 import androidx.recyclerview.widget.GridLayoutManager
 import androidx.recyclerview.widget.LinearLayoutManager
 import org.koin.android.ext.koin.androidApplication
+import org.koin.android.ext.koin.androidContext
 import org.koin.dsl.module
 
 val appModule = module {
     factory {
-        LinearLayoutManager(androidApplication())
+        LinearLayoutManager(androidContext())
```

Comment:
Co do zmiany na `androidContext` to miałem na myśli, żeby zrobić to globalnie w całym module, bo inne zależności mają w zasadzie ten sam przypadek. Sory, że tego nie sprecyzowałem!